Add GeoJSON and DateTime, unitCode and observedAt NGSI-LD support (#843)
telefonicaid/iotagent-node-lib: doc/installationguide.md

    any unexpected error.
-   **singleConfigurationMode**: enables the Single Configuration mode for backwards compatibility (see description in
    the Overview). Default to false.
-   **timestamp**: if this flag is activated:
    -   For NGSIv1/NGSIv2, the IoT Agent will add a `TimeInstant` metadata attribute to all the attributes updated from device information. This flag is overwritten by `timestamp` flag in group or device
    provision.

>     -   With NGSI-LD, the standard `observedAt` metadata attribute is created instead.

Reviewer: ```suggestion
    -   With NGSI-LD, the standard `observedAt` property-of-a-property is created instead.
```

As far as I understand, in NGSI-LD the concept of metadata doesn't exist natively and the right term is property-of-a-property (although I may be wrong).
